### PR TITLE
Update manage-self-service-purchases-admins.md

### PR DESCRIPTION
**Pull request by UKayo**
Below is for self-sevice sign-ups and NOT self-service purchase which is what this page is about. Edited the section to tell there is no bulk way to disable/enable.

"Use PowerShell and Microsoft Entra ID to enable or disable all self-service sign-ups You can use PowerShell commands to change the settings that control self-service sign-ups. To turn off all self-service sign-ups, use the MSOnline PowerShell module to change the MsolCompanySettings setting for AllowAdHocSubscriptions in Microsoft Entra ID. For the steps to turn off self-service sign-ups, see Set MsolCompanySettings."

### CODE REVIEW

File: microsoft-365/commerce/subscriptions/manage-self-service-purchases-admins.md [hunk 1]
```diff
@@ -103,9 +103,9 @@ For more information, see [Use AllowSelfServicePurchase for the MSCommerce Power
 
 <a name='use-powershell-and-azure-ad-to-enable-or-disable-all-self-service-sign-ups'></a>
 
-## Use PowerShell and Microsoft Entra ID to enable or disable all self-service sign-ups
+## To enable or disable all self-service sign-ups
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/manage-self-service-purchases-admins.md]:
```suggestion
## Enable or disable all self-service sign-ups
```

File: microsoft-365/commerce/subscriptions/manage-self-service-purchases-admins.md [hunk 2]
```diff
@@ -103,9 +103,9 @@ For more information, see [Use AllowSelfServicePurchase for the MSCommerce Power
...
 
-You can use PowerShell commands to change the settings that control self-service sign-ups. To turn off all self-service sign-ups, use the **MSOnline** PowerShell module to change the **MsolCompanySettings** setting for **AllowAdHocSubscriptions** in Microsoft Entra ID. For the steps to turn off self-service sign-ups, see [Set MsolCompanySettings](/powershell/module/msonline/set-msolcompanysettings).
+To enable or disable all products, you will need to set the policy on a product basis. There is no way to do this in bulk.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/manage-self-service-purchases-admins.md]:
```suggestion
To enable or disable all products, you need to set the policy on a product basis. There is no way to do this in bulk.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @UKayo, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cmcatee-MSFT** [comment]
Following up with the product team on this.

**Comment by cmcatee-MSFT** [comment]
Section was removed from the article.

**Pull request closed without merging**